### ISSUE TITLE
Versioning confusion

### ISSUE THREAD

**Issue by WTIGER001** [ryankurte/go-mapbox#27]
I know this is my fault but I still need help. I have followed the readme for the Forward geocoding and I am getting compile issues. 

```
import (
	"fmt"
	"gopkg.in/ryankurte/go-mapbox.v0/lib"
	"gopkg.in/ryankurte/go-mapbox.v0/lib/geocode"
)

func GeocodeForward(address string, city string, state string, zip string) (float64, float64, error) {
	// Get the token
	token, err := GetMapboxToken()
	if err != nil {
		return 0, 0, err
	}
	mapBox := mapbox.NewMapbox(token)

	place := fmt.Sprintf("%v %v %v %v", address, city, state, zip)

	var forwardOpts geocode.ForwardRequestOpts
	forwardOpts.Limit = 1
	forward, err := mapBox.Geocode.Forward(place, &forwardOpts)
	if err != nil {
		return 0, 0, err
	}
	
	if len(forward.Features > 0 ) {
		lat := forward.Features[0].Geometry.Coordinates[0]
		lon := forward.Features[0].Geometry.Coordinates[1]
		return lat, lon, nil
	}

	return 0, 0, fmt.Errorf("No Data found")
}
```

I am getting the following error messages: 

`.\mapbox.go:22:48: cannot use &forwardOpts (type *"gopkg.in/ryankurte/go-mapbox.v0/lib/geocode".ForwardRequestOpts) as type *"github.com/ryankurte/go-mapbox/lib/geocode".ForwardRequestOpts in argument to mapBox.Geocode.Forward
.\mapbox.go:27:26: invalid operation: forward.FeatureCollection.Features > 0 (mismatched types []base.Feature and int)`

Also my Go mod file has
`gopkg.in/ryankurte/go-mapbox.v0 v0.4.2`

I can see that it is getting confused between 

gopkg.in/ryankurte/go-mapbox.v0/lib
and 
gopkg.in/ryankurte/go-mapbox/lib

but I do not know how to resolve. Thank you.

**Comment by ryankurte** [comment]
huh, now we have go modules you _probably_ shouldn't need to use `gopkg.in`. i've just added a module file and released `v0.4.3` so, try using `github.com/ryankurte/go-mapbox` everywhere instead?

**Comment by WTIGER001** [comment]
Yup... That worked thanks.
